src/Microsoft.ML.Data/DebuggerExtensions.cs

```diff
@@ -9,7 +9,7 @@
 namespace Microsoft.ML
 {
     /// <summary>
-    /// Static extensions for data preview.
+    /// Class used to create instances of preview objects for debugging.
```

Comment:
>debugging. [](start = 62, length = 10)

please explicitly explain that this class and all the methods here are only used for debugging and should **not** be used in production code.